React/React.xcodeproj/project.pbxproj

```diff
@@ -308,6 +314,10 @@
 				13B080151A69489C00A75B9A /* RCTTextField.m */,
 				13B080161A69489C00A75B9A /* RCTTextFieldManager.h */,
 				13B080171A69489C00A75B9A /* RCTTextFieldManager.m */,
+				BBE9C5BE1AE9D24200D3069B /* RCTTextView.h */,
```

Comment:
Can you add this to the RCTText lib instead of the main React lib? Otherwise it will clash with our private FBTextView implementation.